### PR TITLE
[rawdata2hdf5] Add warning to deprecate rawdata2hdf5

### PR DESCRIPTION
**Pull request by Seunghui98**
This commit adds warning to deprecate rawdata2hdf5.

ONE-DCO-1.0-Signed-off-by: SeungHui Lee <shsh1004.lee@samsung.com>

---
Related Issue: https://github.com/Samsung/ONE/issues/12053
Draft PR: https://github.com/Samsung/ONE/pull/12186

### CODE REVIEW

File: compiler/rawdata2hdf5/rawdata2hdf5 [hunk 1]
```diff
@@ -105,6 +106,7 @@ def create_hdf5(data_list, output_path):
 
 
 def main():
+    warnings.warn("[rawdata2hdf5] section will be deprecated. Please use [one-create-quant-dataset] section.");
```
Comment (seanshpark) [inline compiler/rawdata2hdf5/rawdata2hdf5:109]: Q) what is `[rawdata2hdf5] section` ?

Comment (Seunghui98) [reply compiler/rawdata2hdf5/rawdata2hdf5:109]: @seanshpark 

> Q) what is [rawdata2hdf5] section ?

Simply, it means the tool of `rawdata2hdf5`. 😅
I was thinking about what would be better for this warning message, 
and I used a message referring to this. --- > [reference](https://github.com/Samsung/ONE/blob/c91b55370e28993569059233f744a0085d0dbbea/compiler/one-cmds/onelib/CfgRunner.py#L66)


Is it better to use the word `tool` than the word `section`?

Comment (seanshpark) [reply compiler/rawdata2hdf5/rawdata2hdf5:109]: @jinevening , any comments?

Comment (seanshpark) [reply compiler/rawdata2hdf5/rawdata2hdf5:109]: > and I used a message referring to this. 

do you understand the word `section` from the reference?

Comment (jinevening) [reply compiler/rawdata2hdf5/rawdata2hdf5:109]: @Seunghui98  `rawdata2hdf5` can be called in the command line, so `section` seems not appropriate. Could you update the message as below?

```suggestion
    warnings.warn("rawdata2hdf5 will be deprecated. Please use one-create-quant-dataset.");
```

Comment (Seunghui98) [reply compiler/rawdata2hdf5/rawdata2hdf5:109]: @jinevening 
Thank you for your comment :)

Comment (seanshpark) [reply compiler/rawdata2hdf5/rawdata2hdf5]: FYI `section` came from https://en.wikipedia.org/wiki/INI_file#Sections

**Review by jinevening — APPROVED** [review]
LGTM

**Pull request merged**